assets/js/translator.js

```diff
@@ -147,6 +147,9 @@ if(modeEnabled('translation')) {
             if($('div#translateText').is(':visible')) {
                 translateText();
             }
+            else if($('div#docTranslation').is(':visible')) {
+                translateDoc();
+            }
```

Comment:
Um, I am not sure why this is appearing in the diff here :/ . I simply did a `git rebase` with master. Do you suggest making a separate PR? (The below snippet has the changes that you asked to do @sushain97 )!